src/Components/Aspire.Azure.AI.OpenAI/AspireAzureOpenAIClientBuilder.cs

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.OpenAI;
+using Azure.AI.OpenAI;
+using Microsoft.Extensions.Hosting;
+
+namespace Aspire.Azure.AI.OpenAI;
+
+/// <summary>
+/// A builder for configuring an <see cref="AzureOpenAIClient"/> service registration.
+/// </summary>
+public class AspireAzureOpenAIClientBuilder : AspireOpenAIClientBuilder
```

Comment:
The point of this subclass is just so that, if we want to add **Azure** OpenAI-specific info to the builder in the future, we can do so non-breakingly.